app/models/server.rb

```diff
@@ -40,20 +39,16 @@ class Server < ActiveRecord::Base
   DOMAIN_HLDS = 0
   DOMAIN_HLTV = 1
   DOMAIN_NS2 = 2
-  HLTV_IDLE = 1200
-  DEMOS = "/var/www/virtual/ensl.org/hlds_l/ns/demos"
-  QSTAT = "/usr/bin/quakestat"
-  TMPFILE = "tmp/server.txt"
 
-  attr_accessor :rcon_handle, :pwd
+  attr_accessor :pwd
   attr_protected :id, :user_id, :updated_at, :created_at, :map, :players, :maxplayers, :ping, :version
 
   validates_length_of [:name, :dns,], :in => 1..30
   validates_length_of [:rcon, :password, :irc], :maximum => 30, :allow_blank => true
   validates_length_of :description, :maximum => 255, :allow_blank => true
   validates_format_of :ip, :with => /\A[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\z/
-    validates_format_of :port, :with => /\A[0-9]{1,5}\z/
-    validates_format_of :reservation, :with => /\A[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}:[0-9]{1,5}\z/, :allow_nil => true
+  validates_format_of :port, :with => /\A[0-9]{1,5}\z/
+  validates_format_of :reservation, :with => /\A[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}:[0-9]{1,5}\z/, :allow_nil => true
```

Comment:
Line is too long. [128/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.